randomizer/Lists/KasplatLocations.py

```diff
@@ -388,14 +392,14 @@ def setKasplat(self, state=True):
         KasplatLocation(
             name="Aztec Kasplat: Inside the Llama Temple Matching Game",
             map_id=Maps.AztecLlamaTemple,
-            kong_lst=[Kongs.lanky],
+            kong_lst=[Kongs.donkey, Kongs.diddy, Kongs.lanky, Kongs.tiny, Kongs.chunky],
             coords=[1080, 642, 2240],
             xmin=1009,
             xmax=1161,
             zmin=2130,
             zmax=2332,
             region=Regions.LlamaTemple,
-            additional_logic=lambda l: l.grape or l.phasewalk,
+            additional_logic=lambda l: (l.grape and ((not l.settings.disable_tag_barrels and (l.isdonkey or l.isdiddy or l.istiny or l.ischunky)) or l.islanky)) or l.phasewalk,
```

Comment:
This is only a thing with Tag Anywhere. Tag Anywhere prevents this door closing as a gameplay quality provision. I'd advise reverting this change to what it was before